--- origsrc/libmp3splt-0.7.1/plugins/Makefile.am	2011-10-28 15:16:12.000000000 -0500
+++ src/libmp3splt-0.7.1/plugins/Makefile.am	2011-11-27 23:28:33.010088700 -0600
@@ -8,1 +8,1 @@ plugin_LTLIBRARIES =
-   -no-undefined -lm -lmp3splt @LIBINTL@
+   -no-undefined -module -lm -lmp3splt @LTLIBINTL@
--- origsrc/libmp3splt-0.7.1/src/Makefile.am	2011-05-27 15:20:19.000000000 -0500
+++ src/libmp3splt-0.7.1/src/Makefile.am	2011-11-27 23:24:40.916813800 -0600
@@ -11,1 +11,1 @@ lib_LTLIBRARIES = libmp3splt.la
-libmp3splt_la_LIBADD = @LIBINTL@ -lm
+libmp3splt_la_LIBADD = @LTLIBINTL@ -lm
