--- origsrc/libmp3splt-0.8/doc/Doxyfile.in	2013-01-23 23:20:20.267983600 -0600
+++ src/libmp3splt-0.8/doc/Doxyfile.in	2013-01-23 23:21:31.308186000 -0600
@@ -655,7 +655,7 @@ WARN_LOGFILE           =
 # directories like "/usr/src/myproject". Separate the files or directories
 # with spaces.
 
-INPUT                  = ../src ../include/libmp3splt ../plugins
+INPUT                  = @top_srcdir@/src @top_srcdir@/include/libmp3splt @top_srcdir@/plugins
 #INPUT                  = ../include/libmp3splt/mp3splt.h
 
 # This tag can be used to specify the character encoding of the source files
@@ -891,7 +891,7 @@ HTML_HEADER            =
 # each generated HTML page. If it is left blank doxygen will generate a
 # standard footer.
 
-HTML_FOOTER            = empty_footer.html
+HTML_FOOTER            =
 
 # The HTML_STYLESHEET tag can be used to specify a user-defined cascading
 # style sheet that is used by each HTML page. It can be used to
--- origsrc/libmp3splt-0.8/doc/Makefile.am	2013-01-23 23:20:20.267983600 -0600
+++ src/libmp3splt-0.8/doc/Makefile.am	2013-01-23 21:11:56.000000000 -0600
@@ -20,13 +20,13 @@ endif
 	echo temporary > doxyfile.temp
 
 doxyfile.temp:
-	cat Doxyfile.in >Doxyfile
+	sed -e "s|\@top_srcdir\@|$(top_srcdir)|g" $(srcdir)/Doxyfile.in >Doxyfile
 if HAVE_GRAPHVIZ
 	echo "HAVE_DOT = YES" >> Doxyfile
 else
 	echo "HAVE_DOT = NO" >> Doxyfile
 endif
-	echo "STRIP_FROM_PATH = $(top_builddir)" >> Doxyfile
+	echo "STRIP_FROM_PATH = $(top_srcdir)" >> Doxyfile
 	@echo "Running doxygen to generate the code documentation"
 	doxygen
 	echo temporary > doxyfile.temp
